### ISSUE TITLE
Popover: bug on validation form

### ISSUE THREAD

**Issue by TheNomo3000** [akveo/nebular#1252]
### Issue type

* [x] bug report
* [ ] feature request

### Issue description

**Current behavior:**
The popover, which is used in validation, gets caught and does not go away when the value is valid.

**Expected behavior:**
It is expected that when the value is valid, the popover does not appear.

**Steps to reproduce:**
Implements popover in inputs for validation

**Related code:**

```html
<input
 . . . .
           [nbPopover]="emailValidation"
           [nbPopoverTrigger]="email.invalid && email.touched ? 'hint' : 'noop'"
           nbPopoverPlacement="top"
 . . . . 
/>

    <ng-template #emailValidation >
      <span *ngIf="email.errors?.required">
        Email is required!
      </span>
      <span *ngIf="email.errors?.pattern">
        Email should be the real one!
      </span>
    </ng-template>
```

### Other information:

![image](https://user-images.githubusercontent.com/24535623/53136692-238c6400-3580-11e9-8421-8dae93daf90d.png)

![image](https://user-images.githubusercontent.com/24535623/53136712-37d06100-3580-11e9-9315-874a42b9d2e3.png)

![image](https://user-images.githubusercontent.com/24535623/53136736-4f0f4e80-3580-11e9-99ef-bf55ab713962.png)

**Comment by nnixaa** [comment]
Hi @TheNomo3000, what Nebular version are you on? And also, could you post a reproducible example? You can use https://stackblitz.com/github/akveo/nebular-seed as a starter.

**Comment by TheNomo3000** [comment]
Hi @nnixaa 
    "@nebular/auth": "^3.2.1",
    "@nebular/theme": "^3.2.1",
I will try to create a reproducible example

**Comment by nnixaa** [comment]
Should be fixed in 4.0
